packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -227,10 +228,15 @@ export class CachedModel<
       ]);
     } else {
       dbOperation = Promise.all([
-        this.model.bulkCreate(records, {
-          transaction: tx,
-          updateOnDuplicate: Object.keys(records[0]) as unknown as (keyof T)[], // TODO is this right? we want upsert behaviour
-        }),
+        // cockroach having issue with bulkCreate
```

Comment:
Can we elaborate on this? Is it just the size of the query or something else?

Reply:
Unconfirmed error:
```
(node:6895) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'unique' of undefined
    at recursiveBulkCreate (/Users/jiatwork/ofn/subql/node_modules/sequelize/src/model.js:2782:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

But I suspect this is due to we remove the indexes from model. I need to check this

Reply:
Confirmed this issue due to we remove indexes from model, however after model synced with db, seems like we could not added it back to model.

So I pick use `upsert` here. See https://github.com/subquery/subql/issues/1606